### PR TITLE
disable pain effect in zeus camera

### PR DESCRIPTION
**Pull request by SzwedzikPL**

### CODE REVIEW

**Comment by esteldunedain** [comment]
:+1: We had something almost exactly like this on AGM

File: addons/medical/XEH_postInit.sqf [hunk 1]
```diff
@@ -174,52 +174,54 @@ GVAR(lastHeartBeatSound) = ACE_time;
...
-                    GVAR(effectPainCA) ppEffectAdjust [_strength, _strength, false];
-                    GVAR(effectPainCA) ppEffectCommit 0.01;
-                    [{
-                        GVAR(effectPainCA) ppEffectAdjust [(_this select 0), (_this select 0), false];
-                        GVAR(effectPainCA) ppEffectCommit (_this select 1);
-                    }, [_strength * 0.1, _interval * 0.2], _interval * 0.05, 0] call EFUNC(common,waitAndExecute);
-                    [{
-                        GVAR(effectPainCA) ppEffectAdjust [(_this select 0), (_this select 0), false];
+            // Pain effect, no pain effect in zeus camera
+            if (isNull (findDisplay 312)) then {
```
Comment (SilentSpike) [inline addons/medical/XEH_postInit.sqf]: Do you think it's worth changing this to `isNull curatorCamera` to be consistent throughout ACE?

Comment (esteldunedain) [inline addons/medical/XEH_postInit.sqf]: Yes

Comment (SzwedzikPL) [inline addons/medical/XEH_postInit.sqf]: First i wanted to use curatorCamera but i saw `if (!(isNull (findDisplay 312)) or !(alive ACE_player)) exitWith {` in ace_medical postinit so i used this.

**Comment by esteldunedain** [comment]
Great

**Pull request merged**